internal/ui/widgetpanel.go

```diff
@@ -134,11 +134,8 @@ func (w *widgetPanel) showAccountMenu(from fyne.CanvasObject) {
 		w.desk.WindowManager().Close()
 	}))
 
-	popup := widget.NewPopUpMenu(fyne.NewMenu("Account", items...), root.Canvas())
-
 	bottomLeft := fyne.CurrentApp().Driver().AbsolutePositionForObject(from)
-	popup.Move(bottomLeft.Subtract(fyne.NewPos(0, popup.MinSize().Height)))
-	popup.Resize(fyne.NewSize(from.Size().Width, popup.MinSize().Height))
+	widget.ShowPopUpMenuAtPosition(fyne.NewMenu("Account", items...), root.Canvas(), bottomLeft)
```

Comment:
Don't know if this is the best solution either. It is what I came up with and I think it seems to work.

Reply:
With this change is it still the same width as the widget panel?
What was the error that this change fixes?

Reply:
Sorry for a late answer. It is trying to work around the fact that `widget.NewPopUpMenu` is deprecated. Is this just deprecated in the meaning that it no longer opens the popup on calling it? If so, then I can happily revert this :)

Reply:
That is the case yes.